modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -571,8 +570,8 @@ public boolean stopRaftNodes(ReplicationGroupId groupId) {
     public void destroyRaftNodeStorages(RaftNodeId nodeId, RaftGroupOptions groupOptions) {
         // TODO: IGNITE-23079 - improve on what we do if it was not possible to destroy any of the storages.
         try {
-            String uri = nodeIdStr(nodeId);
-            groupOptions.getLogStorageFactory().destroyLogStorage(uri);
+            String logUri = nodeIdStr(nodeId);
```

Comment:
What's this change for?

Reply:
The term seems to be a bit more accurate